enos/enos-scenario-e2e-aws-windows.hcl

```
@@ -0,0 +1,312 @@
+  step "create_boundary_cluster" {
+    module = module.aws_boundary
+    depends_on = [
+      step.create_base_infra,
+      step.create_windows_client,
+      step.create_db_password,
+      step.build_boundary,
+      step.create_vault_cluster,
+      step.read_boundary_license
+    ]
+
+    variables {
+      boundary_binary_name        = var.boundary_binary_name
+      boundary_install_dir        = local.boundary_install_dir
+      boundary_license            = var.boundary_edition != "oss" ? step.read_boundary_license.license : null
+      common_tags                 = local.tags
+      controller_instance_type    = var.controller_instance_type
+      controller_count            = var.controller_count
+      db_pass                     = step.create_db_password.string
+      kms_key_arn                 = step.create_base_infra.kms_key_arn
+      local_artifact_path         = step.build_boundary.artifact_path
+      ubuntu_ami_id               = step.create_base_infra.ami_ids["ubuntu"]["amd64"]
+      vpc_id                      = step.create_base_infra.vpc_id
+      worker_count                = var.worker_count
+      worker_instance_type        = var.worker_instance_type
+      controller_config_file_path = "templates/controller_bsr.hcl"
+      worker_config_file_path     = "templates/worker_bsr.hcl"
+      vault_address               = step.create_vault_cluster.instance_public_ips[0]
+      vault_transit_token         = step.create_vault_cluster.vault_transit_token
+      aws_region                  = var.aws_region
+      ip_version                  = "dual"
+      recording_storage_path      = "/recording"
+      alb_sg_additional_ips       = step.create_windows_client.public_ip_list
+    }
+  }
+
+  step "create_test_id" {
+    module = module.random_stringifier
+    variables {
+      length = 5
+    }
+  }
+
+  step "iam_setup" {
+    module = module.aws_iam_setup
+    depends_on = [
+      step.create_base_infra,
+      step.create_test_id
+    ]
+
+    variables {
+      test_id    = step.create_test_id.string
+      test_email = var.test_email
+    }
+  }
+
+  step "create_bucket" {
+    module = module.aws_bucket
+    depends_on = [
+      step.create_boundary_cluster,
+    ]
+    variables {
+      cluster_tag = step.create_boundary_cluster.cluster_tag
+      user        = step.iam_setup.user_name
+      is_user     = true
+    }
+  }
+
+  step "create_rdp_server" {
+    module = module.aws_rdp_server
+    depends_on = [
+      step.create_base_infra,
+    ]
+
+    variables {
+      vpc_id         = step.create_base_infra.vpc_id
+      server_version = matrix.rdp_server
+    }
+  }
+
+  step "run_e2e_test" {
+    module = module.test_e2e
+    depends_on = [
+      step.create_boundary_cluster,
+      step.create_rdp_server,
+      step.create_bucket
+    ]
+
+    variables {
+      test_package             = "github.com/hashicorp/boundary/testing/internal/e2e/tests/base"
+      debug_no_run             = var.e2e_debug_no_run
```

It might make sense to hard-code this to true so it never automatically runs these tests directly. 
```suggestion
      debug_no_run             = true
```

---

Good idea. Changed